homeassistant/components/spider/climate.py

```diff
@@ -43,6 +43,13 @@ def __init__(self, api, thermostat):
         """Initialize the thermostat."""
         self.api = api
         self.thermostat = thermostat
+        self.support_fan = thermostat.fan_speed_values
+        self.support_hvac = []
+        for operation_value in thermostat.operation_values:
+            if operation_value in SPIDER_STATE_TO_HA:
+                self.support_hvac.append(SPIDER_STATE_TO_HA.get(operation_value))
```

Comment:
We already determined that the key exists, we can just do
```suggestion
                self.support_hvac.append(SPIDER_STATE_TO_HA[operation_value])
```